src/logic/calculate.js

```diff
@@ -0,0 +1,97 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
```

Comment:
`isNumber(item)` should not be here in case you want to include this functionality to your implementation create a helper file to store them since calculate,js should be the only function in the module.
Personally I don't think `isNumber(item)` is needed, users are going to be selecting values from the calculator you gave them since they don't have the flexibility of inputting what they want but the values of the calculator, checking whether they selected a number is not needed. The user has two options number selection or operation selection. hence `operation.js` and `calculate.js` should be enough.

Reply:
here I'm checking for three things: 
1- if the btn is AC.
2- if the btn is a number, hence the isNumber function.
3- if the btn is "=".
4- if the btn is none of the above which in this case is an operator.